When running an instrumented build, test_asyncio takes an incredibly long
time (and currently fails). Skip it.

diff -wpruN '--exclude=*.orig' a~/Makefile.pre.in a/Makefile.pre.in
--- a~/Makefile.pre.in	1970-01-01 00:00:00
+++ a/Makefile.pre.in	1970-01-01 00:00:00
@@ -258,7 +258,7 @@ TCLTK_LIBS=	@TCLTK_LIBS@
 # The task to run while instrumented when building the profile-opt target.
 # We exclude unittests with -x that take a rediculious amount of time to
 # run in the instrumented training build or do not provide much value.
-PROFILE_TASK=-m test.regrtest --pgo
+PROFILE_TASK=-m test.regrtest --pgo -x test_asyncio
 
 # report files for gcov / lcov coverage report
 COVERAGE_INFO=	$(abs_builddir)/coverage.info
